packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -531,6 +583,26 @@ $Calendar-dayPicker-margin: 10px;
     .monthPicker {
       display: block;
     }
+
+  }
+
+  // Do not hide Month picker if it is the only calendar
+  .pickerOnlyMonthPickerVisible, .overlayedCalendars{
```

Comment:
When @media (max-width: 459px) all the monthpickers are set to display none.  This prevents the month pickers that need to be visible from disappearing.  The "Inline Calendar with month picker and no day picker" in the example is styled with .pickerOnlyMonthPickerVisible, and should not be set to display: none  at @media (max-width: 459px).  The overlayed callendars (styled with .overlayedCalendars) also should not have a month picker that is set to display:none at @media (max-width: 459px).

![calendar1](https://user-images.githubusercontent.com/25463697/29325909-5416d444-819e-11e7-962b-d5936128ff74.jpeg)